content-src/activity-stream.jsx

```diff
@@ -19,8 +19,10 @@ new DetectUserSessionStart(store).sendEventOrAddListener();
 // to request state rehydration (see Base.jsx). If we are NOT in a prerendered state,
 // we can request it immedately.
 if (!global.gActivityStreamPrerenderedState) {
+  console.log("Prerendered html not used for Discovery Stream New Tab");
```

Comment:
Added comment to verify in console for non-prerendered version for discovery stream new tab